### PR TITLE
Integrating unito

### PR DESCRIPTION
**Pull request by xeniadg**
Adding Unito to the marketplace.

### CODE REVIEW

**Comment by xeniadg** [comment]
Hey Javan,

Thanks for getting back to me! I agree that the usual way you guys list integrations is to have one listing for the provider. I think that’s not the right approach for Unito though. Because we integrate work management tools Asana, GItHub, JIRA, Trello, and Wrike, we unlock the ability for Basecamp to find users from the 50+ million people that are already using those tools to manage their work. But it’s important that they be able to find us. If we list under Unito, basically no one will find the listing and your efforts and mine are largely wasted. If we list our sync integration under the software that people want to integrate, that’s when we’re findable and both Basecamp and Unito can win.

By way of example, [literally](https://marketplace.atlassian.com/search?query=unito) [every](https://info.trello.com/power-ups/unito) [other](https://asana.com/apps) [marketplace](https://www.wrike.com/apps/) we’ve integrated with lists each of the applications that we sync separately. I hope you will, too. :)

Cheers,

Xenia

**Comment by javan** [comment]
Sorry, but you can only add one entry for your own company.

> If we list under Unito, basically no one will find the listing 

You're free to list all of the services you tie together in your entry's description. I think we you wrote above is clear and plenty findable:

> we integrate work management tools Asana, GItHub, JIRA, Trello, and Wrike, we unlock the ability for Basecamp to find users from the 50+ million people that are already using those tools to manage their work.

**Comment by xeniadg** [comment]
Hey Javan, I've updated the submission to only one Unito listing.

**Comment by xeniadg** [comment]
Hey @javan , I just wanted to follow up with you about Unito's listing on Basecamp's integration page. We have submitted the MR 2 weeks ago, and would like to know when it will be live on the integration page? :)

**Comment by xeniadg** [comment]
@javan I've removed the two .DS_Store files, please approve :)

File: integrations.yml [hunk 1]
```diff
@@ -106,7 +111,7 @@
   description: "Connect Basecamp with 50+ apps. Slack, Gmail, Trello, Google Calendar, Google Sheets, Wufoo, Pipedrive and more."
   website: "https://automate.io/integration/basecamp-3"
   icon: "automate.io.png"
-  category: 3
+  category: 1
```
Comment (javan) [inline integrations.yml]: Accidental change?

**Pull request closed without merging**